app/components/ui/LabelTag.tsx

```diff
@@ -2,18 +2,26 @@ import React from "react";
 import classNames from "classnames";
 import styles from "./LabelTag.module.scss";
 
-interface LabelTagProps {
-  label: string;
-  withTooltip?: boolean;
-}
+// This component relies on the shape and naming conventions of data Shelter
+// Tech syncs to Algolia which distinguishes service and organization results
+// with a "type" field.
+//
+// CAVEAT: It's unlikely this will change without our knowledge but developers
+// should be aware of the possibility.
+const SHELTER_TECH_TYPE_FOR_ORGANIZATION = "resource";
 
-export const LabelTag = (props: LabelTagProps) => {
-  const { label, withTooltip = false } = props;
+export const LabelTag = ({ label }: { label: string }) => {
```

Comment:
**issue (blocking):** `LabelTag` was a reusable component before, and it is now specific to the service card. So on the service listing page, the `LabelTagRows` component, which uses `LabelTag` is now displaying nothing.